src/reducers/posts.js

```diff
@@ -0,0 +1,28 @@
+import { GET_POSTS, GET_USER, GET_POST, GET_COMMENTS_BY_ID, UPDATE_SEARCH_TERM } from '../actions/posts';
+
+
+const initialState = {
+  posts: [],
+  comments: [],
+  post: {},
+  user: {},
+  searchTerm: 'x'
```

Comment:
```suggestion
  searchTerm: ''
```
For user experience, would be better to start with an empty string -- that way all posts load on /posts. If you run into errors, maybe it's time to look into a guard clause for whoever needs searchTerm.